src/components/Loader/PageNavLink.tsx

```diff
@@ -0,0 +1,20 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+import { NavLink } from 'react-router-dom';
+
+type Props = {
+  to: string;
+  text: string;
```

Comment:
Better to give meaningful names
```suggestion
  url: string;
  title: string;
```

Reply:
changed